doc/nvim-tree-lua.txt

```diff
@@ -1,90 +1,125 @@
 *nvim-tree.lua* A file explorer written in lua
 
-Minimum version of neovim: 0.6.0
-
 Author: Yazdani Kiyan <yazdani.kiyan@protonmail.com>
 
 ==============================================================================
-INTRODUCTION                                         *nvim-tree-introduction*
+CONTENTS                                                          *nvim-tree
 
-This file explorer requires `neovim >= 0.6.0`
+  1. Introduction                             nvim-tree-introduction
+  2. Quickstart                               nvim-tree-quickstart
+  3. Commands                                 nvim-tree-commands
+  4. Setup/Configuration                      nvim-tree-setup
+  5. Mappings                                 nvim-tree-mappings
+  6. Highlight Groups                         nvim-tree-highlight
+  7. Events                                   nvim-tree-events
 
 ==============================================================================
-QUICK START                                          *nvim-tree-quickstart*
+ 1. INTRODUCTION                                     nvim-tree-introduction
 
-open the tree with :NvimTreeToggle
->
-  nnoremap <C-n> :NvimTreeToggle<CR>
-  lua require 'nvim-tree'.toggle()
-<
+<add shortish description here>
+
+Features:
+    File icons with vim-devicons.
+
+    Uses other type of icons so a good font support is recommended.
+    If the tree renders weird glyphs, install the correct fonts.
```

Comment:
> I like this too, this is a good change. We should also update the `update-default-opts` script to avoid writing to readme.

Done in last commit but did not test it at all.

> @gegoune tell us if you want us to finish this PR, or if you are willing to do so yourself :)

Either way - I don't mind continuing to work on that but maintainers have edits writes to this pr so feel free to do whatever. :)

Reply:
Pushed https://github.com/kyazdani42/nvim-tree.lua/pull/1327/commits/84a1b9804b1831a046471b86062ab0ceb11acc22